### ISSUE TITLE
Feature Request: Select object on click and temporarily highlight it

### ISSUE THREAD

**Issue by IAmMarcelJung** [FPGA-Research-Manchester/FABulator#3]
Hi, it would be nice if objects could be selected by clicking the left mouse button (LMB). They could then be temporarily highlighted (e.g. until something different is selected or the user just clicks somewhere else) so it is a few click less to see e.g. where the wire ends. IMHO it would also feel more responsive if you get some feedback when clicking the LMB. Maybe also the tooltip could be shown on click so it appears faster, but this should be considered after seeing it in action. Let me know what you think about it! :smiley:

**Comment by JakobTernes** [comment]
Sounds like a good idea. In addition, what do you think about some sort of highlighting while hovering over wires (or other elements)?

**Comment by IAmMarcelJung** [comment]
Yeah this would also be a possibility! However it could also be considered too "aggressive" since e.g. the wires would then all shortly flash if you hover over a series of them. Nevertheless I think this is something one could play around with! Maybe just a different gray scale could be used for hovering and clicking would then set a totally different color? As I once already mentioned it would be nice to let the user configure it depending on personal preference, so it could also be completely turned off, but I guess this is for another issue then...

**Comment by JakobTernes** [comment]
Making it configurable should be little extra effort as soon as it is implemented - I think thats a good idea too.
